library/agent/logger/logLevel.ts

```diff
@@ -0,0 +1,30 @@
+import { isDebugging } from "../../helpers/isDebugging";
+
+export enum AikidoLogLevel {
+  debug = 1,
+  info = 2,
+  warn = 3,
+  error = 4,
+}
+
+const logLevelKeys = Object.keys(AikidoLogLevel);
+const defaultLogLevel = AikidoLogLevel.info;
```

Comment:
Not sure if default level should be `info` or `warn`.

`Zen has detected a NoSQL injection ...` is logged as info.